circle.yml

```diff
@@ -23,9 +23,9 @@ dependencies:
        0)
          sudo apt-get install language-pack-it && ./ci/install_circle.sh JOB="2.7_COMPAT" LOCALE_OVERRIDE="it_IT.UTF-8" ;;
        1)
-         sudo apt-get install language-pack-zh-hans && ./ci/install_circle.sh JOB="3.4_SLOW" LOCALE_OVERRIDE="zh_CN.UTF-8" ;;
+         sudo apt-get install language-pack-zh-hans && ./ci/install_circle.sh JOB="3.6_SLOW" LOCALE_OVERRIDE="zh_CN.UTF-8" ;;
```

Comment:
actuall let's name these

3.6_LOCALE_SLOW
3.6_LOCALE

I think it more consistent